src/filters.php

```diff
@@ -32,11 +29,28 @@
 });
 
 /**
- * Use theme wrapper
+ * Template Hierarchy should search for .blade.php files
  */
-add_filter('template_include', function ($main) {
-    if (!is_string($main) && !(is_object($main) && method_exists($main, '__toString'))) {
-        return $main;
-    }
-    return ((new Template(new Wrapper($main)))->layout());
-}, 109);
+array_map(function ($tag) {
+    add_filter("{$tag}_template_hierarchy", function ($templates) {
+        return array_merge(str_replace('.php', '.blade.php', $templates), $templates);
+    });
+}, [
+    'index', '404', 'archive', 'author', 'category', 'tag', 'taxonomy', 'date', 'home',
+    'front_page', 'page', 'paged', 'search', 'single', 'singular', 'attachment'
```

Comment:
`front_page` should be corrected to `frontpage` https://github.com/WordPress/WordPress/blob/4.7-branch/wp-includes/template.php#L29-L41